### ISSUE TITLE
ADA - Not enough color contrast

### ISSUE THREAD

**Issue by katelyndavis** [immregistries/mqe#79]
[Overview]
All text elements must have sufficient contrast between text in the foreground and background colors behind it in accordance with WCAG 2 AA contrast ratio thresholds.

Maybe need to figure out a new default theme?

[Requirements]
[https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=AxeChrome](url)

[Dev Notes]
The red circled text are hard violations. The blue were labeled as needs review....they seem to be the same color as the red ones so i don't know why they got flagged as less serious. I assume that changing the theme will fix them all.
![image](https://user-images.githubusercontent.com/37371748/42708032-fc6f7b12-86a9-11e8-8580-8160afd25ab9.png)

[Testing]
The attached link has a color contrast tool. You can enter the foreground and background color codes to see if they pass
